types/recharts/index.d.ts

```diff
@@ -735,16 +735,20 @@ export interface TreemapProps {
 	animationEasing?: AnimationEasingType;
 }
 
-export interface Label {
+export interface LabelProps {
 	viewBox?: ViewBox | PolarViewBox;
 	formatter?: LabelFormatter;
-	value: string | number;
+	value?: string | number;
 	position?: PositionType;
 	offset?: number;
 	content?: React.ReactElement<any> | ContentRenderer<Label>;
 	children?: any;
+	angle?: number;
+	style?: React.CSSProperties;
```

Comment:
I'm not exactly sure about the style attribute. Is this not part of the `React.Component`?

Reply:
Good question. I'm not sure either.

Reply:
Looks like a standard `React.Component` only gets `children` added by default: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/v15/index.d.ts#L299

At least as of React 15, [`ReactHTMLElement`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/v15/index.d.ts#L119) gets a `style` prop via `AllHTMLAttributes` (and similar for `ReactSVGElement`), but that doesn't apply here.

Reply:
OK, so if `React.Component` only gets `children`, it would make sense to add `style?` here for the `LabelProps` interface.